src/components/Sections/Toolbar/Toolbar.jsx

```diff
@@ -0,0 +1,181 @@
+                <DashboardIcon />
+              </StyledListItemIcon>
+              <ListItemText
+                primary="Overview"
+                primaryTypographyProps={{
+                  fontSize: '.9rem',
+                  fontWeight: 'medium',
+                  lineHeight: '20px',
+                  mb: '2px',
+                }}
+                sx={{ my: 0 }}
+              />
+            </ListItemButton>
+            <Box>
+              <ListItemButton
+                onClick={() => openMenu(menu.ROADMAP)}
+                sx={{
+                  py: '2px',
+                }}
+              >
+                <ListItemText
+                  primary="Roadmap"
+                  primaryTypographyProps={{
+                    fontSize: '.8rem',
+                    lineHeight: '20px',
+                    mb: '2px',
+                    color: 'grey.700',
+                    fontWeight: 'medium',
+                  }}
+                  sx={{ my: 0 }}
+                />
+                <KeyboardArrowDown
+                  sx={{
+                    mr: -1,
+                    opacity: 1,
+                    transform: open.includes(menu.ROADMAP)
+                      ? 'rotate(-180deg)'
+                      : 'rotate(0)',
+                    transition: '0.2s',
+                  }}
+                />
+              </ListItemButton>
+              {open.includes(menu.ROADMAP) &&
+                roadMapSubFields.map((item) => (
+                  <ListItemButton
+                    key={item.value}
+                    sx={{ py: 0, minHeight: 50, fontWeight: 'medium' }}
+                    component={Link}
+                    to={item.target}
+                  >
+                    <StyledListItemIcon>{item.icon}</StyledListItemIcon>
+                    <ListItemText
+                      primary={item.label}
+                      primaryTypographyProps={{
+                        fontWeight: 'medium',
+                        fontSize: '.9rem',
+                      }}
+                    />
+                  </ListItemButton>
+                ))}
+            </Box>
+          </Grid>
+          <Grid
+            item
+            xs={12}
+            sx={{ height: 'fit-content', alignSelf: 'flex-end' }}
+          >
+            <Divider />
+            <ListItemButton
+              sx={{
+                py: 2,
+              }}
+              component={Link}
+              to="/manage-project"
+            >
+              <StyledListItemIcon>
+                <SettingsIcon />
+              </StyledListItemIcon>
+              <ListItemText
+                primary="Manage project"
+                primaryTypographyProps={{
+                  fontSize: '.9rem',
+                  fontWeight: 'medium',
+                  lineHeight: '20px',
+                  mb: '2px',
+                }}
+                sx={{ my: 0 }}
+              />
+            </ListItemButton>
```

Comment:
Becare of Footer pass through the button

Reply:
You made a comment but you merged it? Why not 😂